docs/develop/create-and-debug-office-add-ins-in-visual-studio.md

```diff
@@ -1,122 +1,145 @@
 
-You can add documents to the add-in project. If you have a document that contains test data that you want to use with your add-in, Visual Studio opens that document for you when you start the project.
+### Use an existing document to debug the add-in
 
-### To use an existing document to debug the add-in
+If you have a document that contains test data you want to use while debugging your Excel, PowerPoint, or Word add-in, Visual Studio can be configured to open that document when you start the project. To specify an existing document to use while debugging the add-in, complete the following steps.
 
-1. In  **Solution Explorer**, choose the add-in project folder.
+1. In **Solution Explorer**, choose the add-in project (*not* the web application project).
     
-    > [!NOTE]
-    > Choose the add-in project and not the web application project.
-
-2. On the  **Project** menu, choose **Add Existing Item**.
+2. From the menu bar, choose **Project** > **Add Existing Item**.
     
-3. In the  **Add Existing Item** dialog box, locate and select the document that you want to add.
+3. In the **Add Existing Item** dialog box, locate and select the document that you want to add.
     
-4. Choose the  **Add** button to add the document to your project.
+4. Choose the **Add** button to add the document to your project.
     
-5. In  **Solution Explorer**, choose the add-in project folder.
-6. On the menu bar, choose  **View** > **Properties Window**.
-7. In the properties window, choose the **Start Document** list, and then choose the document that you added to the project. Now the project is configured to start your add-in in your existing document.
+5. In **Solution Explorer**, choose the add-in project (*not* the web application project).
+
+6. From the menu bar, choose **View** > **Properties Window**.
 
-### Start the solution
+7. In the **Properties** window, choose the **Start Document** list, and then select the document that you added to the project. The project is now configured to start the add-in in that document.
 
-Start the solution from the menu bar by choosing **Debug** > **Start Debugging**. Visual Studio will automatically build the solution and start Office to host your add-in.
+### Start the project
+
+Start the project by choosing **Debug** > **Start Debugging** from the menu bar. Visual Studio will automatically build the solution and start Office to host your add-in.
+
+> [!NOTE]
+> When you start an Outlook add-in project, you'll be prompted for login credentials. If you're asked to log in repeatedly then Basic Auth may be disabled for accounts on your Office 365 tenant. In this case, try using a Microsoft account instead.
 
 When Visual Studio builds the project it performs the following tasks:
 
```

Comment:
I just noticed that in the step 1, it says the manifest is copied to the ProjectName\Output directory. But when I run VS, it puts the manifest in bin\debug\OfficeappManifests folder. Need to confirm, but this info might be out of date.

Reply:
Good catch! I've updated this section accordingly.